src/pages/login.js

```diff
@@ -1,16 +1,26 @@
-import {View, StyleSheet} from 'react-native';
-import React, {Component, PropTypes} from 'react';
-import {routes} from '../routes';
-import core_styles from '../styles/core-styles';
-import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
-import {Sae} from 'react-native-textinput-effects';
-import {Button} from 'react-native-material-ui';
-import theme from '../styles/ui-theme';
+import React, { Component, PropTypes } from 'react';
```

Comment:
I usually organize it like that, react dependencies and then white space, then 3rd libraries and white space and finally our dependencies.